### ISSUE TITLE
HOCONConverter doesn't output compact hocon format

### ISSUE THREAD

**Issue by cryptofred** [chimpler/pyhocon#117]
When using tool.py on the following example hocon file

```
databases.ips = [
  192.168.0.1 // use white space or comma as separator
  "192.168.0.2" // optional quotes
  192.168.0.3, # can have a trailing , which is ok
]
```

it outputs:

```
databases {
  ips = [
    "192.168.0.1"
    "192.168.0.2"
    "192.168.0.3"
  ]
}
```

I specifically mean the following part:
```
databases.ips = [
```
compared to this one:
```
databases {
  ips = [
```

I like hocon for it's readability and being a compact format in comparison to JSON.
It there any option to output the more compact format? Shouldn't it be the default?

Also thanks for the great work.

**Comment by cryptofred** [comment]
Another example:
```
d = collections.OrderedDict()
  d['banana'] = 3
  d['apple'] = 4
  d['pear'] = 1
  d['orange'] = collections.OrderedDict()
  d['orange']['blue'] = 2
  
l = { 'key': d }
config = ConfigFactory.from_dict(l)
print(HOCONConverter.convert(config, 'hocon'))
```

outputs:
```
key {
  banana = 3
  apple = 4
  pear = 1
  orange {
    blue = 2
  }
```

Where I would expect:
```
key {
  banana = 3
  apple = 4
  pear = 1
  orange.blue = 2
}
```

I expect pyhocon leveraging all advantages in hocon output over json output by default or at least provide an option to do so.

What do you think?

**Comment by darthbear** [comment]
Sorry for the long delay in getting back to you. Thanks @cryptofred. It was released in  `pyhocon==0.3.18`.
